### PR TITLE
Add eaas-provision-space task

### PR DESCRIPTION
**Pull request by amisstea**
This task is meant to be used before all other EaaS tasks/stepactions in a pipeline (such as #1058).

### CODE REVIEW

**Comment by amisstea** [comment]
@oamsalem @yftacherzog please review.

**Review by yftacherzog — APPROVED** [review]
lgtm

**Comment by amisstea** [comment]
@konflux-ci/build-maintainers can I please get your approval on this PR?

**Review by chmeliik — APPROVED** [review]
Don't have the context for EaaS, but the code looks reasonable :+1:

**Comment by amisstea** [comment]
@chmeliik, if you have the permission, can you please merge this?

**Pull request merged**
